app/Http/Controllers/Form/WidgetSettingsController.php

```diff
@@ -45,7 +45,10 @@ public function update(Request $request, $widget_settings)
             ]);
         }
 
+        $refresh=(!is_numeric($widget_settings["refresh"]) || (int)$widget_settings["refresh"] < 0 || (int)$widget_settings["refresh"] > 32767 ? 60 : $widget_settings["refresh"]);
```

Comment:
This should be done in a validation rule above.

Reply:
'settings.refresh' => 'int|min:1|max:86400',

Reply:
ok, thanks. but I can't understand - I set not int value and after pressing save button I see empty red alert in up right corner. what I haven't do yet?